source/security/ddos/safedns-setup.md

```diff
@@ -8,7 +8,7 @@ To use DDoSx<sup>®</sup>, CDN and WAF, you need to either have your domains' se
 
 ```eval_rst
 .. warning::
-   This documentation is for setting up DDoSx using SafeDNS. Documentation on setting up an external domain via an ALIAS, ANAME or CNAME can be found in our** [Setup DDoSx via ALIAS, ANAME or CNAME](/security/ddos/setup-cname) documentation.
+   This documentation is for setting up DDoSx using SafeDNS. Documentation on setting up an external domain via an ALIAS, ANAME or CNAME can be found in our [Setup DDoSx via ALIAS, ANAME or CNAME](/security/ddos/setup-cname) documentation.
```

Comment:
Check this about hyperlinks in `eval_rst` blocks: https://github.com/ukfast/docs.ukfast.co.uk#hyperlinks